code/Items/Equipment/Hands/Hands.GrabbableProp.cs

```diff
@@ -6,10 +6,10 @@ namespace TTT;
 public class GrabbableProp : IGrabbable
 {
 	private ModelEntity GrabbedEntity { get; set; }
-	public string PrimaryAttackHint => GrabbedEntity.IsValid() ? "Throw" : string.Empty;
-	public string SecondaryAttackHint => GrabbedEntity.IsValid() ? "Drop" : string.Empty;
+	public virtual string PrimaryAttackHint => GrabbedEntity.IsValid() ? "Throw" : string.Empty;
+	public virtual string SecondaryAttackHint => GrabbedEntity.IsValid() ? "Drop" : string.Empty;
```

Comment:
All the virtuals in this class can be removed since we got rid of `DetectiveHat.cs`